internal/cli/usercmd/service.go

```diff
@@ -64,7 +81,10 @@ func (c *EpinioClient) ServiceCreate(catalogServiceName, serviceName string) err
 	log.Info("start")
 	defer log.Info("return")
 
-	c.ui.Note().Msg("Creating Service...")
+	c.ui.Note().
+		WithStringValue("Class", catalogServiceName).
```

Comment:
I think we should stick to "catalog service" and avoid introducing the "class" term to keep things consistent across the code base

Reply:
Done, commit 6eef435b05a98b2f575a66cf237529c070697ba9